src/app/portlets/dot-edit-content/dot-edit-content.component.html

```diff
@@ -1,3 +1,7 @@
+<dot-edit-page-toolbar class="dot-edit__toolbar" [disabled]="!originalValueChanged" (save)="saveContent()">
```

Comment:
The ```disabled``` attribute it's not a good name here, looks like your are disabling the whole toolbar when is only the save button, so maybe an attr called ```canSave``` it's a better fit here.